### ISSUE TITLE
UI Loads to a blank grey screen

### ISSUE THREAD

**Issue by fake-name** [turboderp/exui#55]
Effectively the title. 

It turns out this

> browser-based UI for running **local** inference 

depends on being able to fetch `https://cdn.jsdelivr.net/npm/marked/marked.min.js`, which does not work without internet access.

Please don't depend on random CDNs for content. Especially for something where the entire purpose is not using web-services.

**Comment by turboderp** [comment]
You're right, and this was actually more of an oversight than anything. Spinning a lot of plates here. I've included a copy of MarkedJS with the latest commit, so it should be able to run with no internet access now.
